src/darker/tests/test_linting.py

```diff
@@ -567,3 +580,35 @@ def test_get_messages_from_linters_for_baseline(git_repo):
         MessageLocation(a_py, 3): [LinterMessage("python", "Third line")],
     }
     assert result == expect
+
+
+class AssertEmptyStderrPopen(Popen[str]):
+    """A Popen to use for the following test; asserts that its stderr is empty"""
+    def __init__(
+        self,
+        args: List[str],
+        stdout: int,
+        encoding: str,
+        cwd: Path,
+        env: Dict[str, str],
+    ):
+        super().__init__(
+            args, stdout=stdout, stderr=PIPE, encoding=encoding, cwd=cwd, env=env
+        )
```

Comment:
Not sure if it fits well with your prevailing style (or typechecking - I'm still learning how to bend mypy to my will :p), but I think you could do

```suggestion
    def __init__(self, args: List[str], **kwargs):
        kwargs['stderr'] = PIPE
        super().__init__(args, **kwargs)
```

(this works for Popen because it explicitly declares its trailing parameters to be keyword-only)

Reply:
I originally had `*args, **kwargs` but Mypy wasn't happy for it. The options were to omit types and do a `#type: ignore`, or to separate and annotate the keyword arguments.

I guess for unit tests and such a minimal helper skipping type checking would be just fine. Would you prefer that?

Reply:
Actually this was trivial to do. The class is now much more concise.